README.md

```diff
@@ -79,6 +79,11 @@ You might also need to provide your DigitalOcean access token in
 the cloud controller to start, but in that case, you will not be able to
 validate integration with DigitalOcean API.
 
+The `DO_IP_ADDR_FAMILIES` is used to configure the required IP familes and the
+order in which address should be populated in nodes status. The accepted values
+are one of the `{"", "ipv4", "ipv6", "ipv4,ipv6", "ipv6,ipv4"}`.IPv4 is the 
```

Comment:
Why do we have curly braces here? It almost looks like JSON but we don't require JSON, do we?

Can we rephrase this in a more user friendly manner, e.g., say `one of "ipv4", "ipv6", or a comma-separated list of multiple IP address families`?

Reply:
fixed